docs/guide/structure-controllers.md

```diff
@@ -420,25 +420,25 @@ to fulfill the request:
    * If the action ID is found to match an action method, an inline action will be created;
    * Otherwise an [[yii\base\InvalidRouteException]] exception will be thrown.
 3. The controller sequentially calls the `beforeAction()` method of the application, the module (if the controller
-   belongs to a module) and the controller.
-   * If one of the calls returns false, the rest of the uncalled `beforeAction()` will be skipped and the
+   belongs to a module), and the controller.
+   * If one of the calls returns false, the rest of the uncalled `beforeAction()` methods will be skipped and the
      action execution will be cancelled.
    * By default, each `beforeAction()` method call will trigger a `beforeAction` event to which you can attach a handler.
-4. The controller runs the action:
-   * The action parameters will be analyzed and populated from the request data;
+4. The controller runs the action.
+   * The action parameters will be analyzed and populated from the request data.
 5. The controller sequentially calls the `afterAction()` method of the controller, the module (if the controller
-   belongs to a module) and the application.
+   belongs to a module), and the application.
    * By default, each `afterAction()` method call will trigger an `afterAction` event to which you can attach a handler.
 6. The application will take the action result and assign it to the [response](runtime-responses.md).
 
 
 ## Best Practices <span id="best-practices"></span>
 
-In a well-designed application, controllers are often very thin with each action containing only a few lines of code.
+In a well-designed application, controllers are often very thin, with each action containing only a few lines of code.
 If your controller is rather complicated, it usually indicates that you should refactor it and move some code
 to other classes.
 
-In summary, controllers
+Here are some specific best practices. Controllers
```

Comment:
This list used to be labeled as a summary, but as far as I can tell it's not a summary of anything on the page. Since it's in the Best Practices section, I assumed it was a list of best practices. But I may well have misunderstood. I'd be happy to replace that line with a better description of what this is a list of.